doc/source/contributing_code.rst

```diff
@@ -515,6 +516,20 @@ Or if you're ok with the suggested formatting changes, just apply the suggested
     darker -r upstream/develop testsuite/MDAnalysisTests -L flake8
 
 
+.. note::
+
+   MDAnalysis does *not* currently use the popular `black`_ code formatter across the whole project, therefore please consider the following rules:
+
+   * Do **not run ``black`` on an existing file** that you are editing. This makes the diff for the PR very difficult to read. Instead use ``darker`` as explained above to only reformat your changes.
```

Comment:
Some interaction with bolding makes the backticks not render properly, unfortunately.

```suggestion
   * **Do not run** ``black`` on an existing file that you are editing. This makes the diff for the PR very difficult to read. Instead use ``darker`` as explained above to only reformat your changes.
```